dts/bindings/pinctrl/ite,it8xxx2-pinctrl.yaml

```diff
@@ -0,0 +1,94 @@
+    use this node to route UART1 RX and TX setting the alternate
+    function on the pin.
+
+    The node has the 'pinctrl' node label set in your SoC's devicetree,
+    so you can modify it like this:
+
+      &pinctrl {
+              /* your modifications go here */
+      };
+
+    All device pin configurations should be placed in child nodes of the
+    'pinctrl' node, as shown in this example:
+
+      /* You can put this in places like a board-pinctrl.dtsi file in
+       * your board directory, or a devicetree overlay in your application.
+       */
+
+      /* include pre-defined pins and functions for the SoC used by the board */
+      #include <dt-bindings/pinctrl/it8xxx2-pinctrl.h>
+
+      &pinctrl {
+        /* configuration for I2C0 default state */
+        i2c0_clk_default: i2c0_clk_default {
+                pinmuxs = <&pinctrlb 3 IT8XXX2_ALT_FUNC_1>;
+                gpio-voltage = "1p8";
+        };
+        i2c0_data_default: i2c0_data_default {
+                pinmuxs = <&pinctrlb 4 IT8XXX2_ALT_FUNC_1>;
+                gpio-voltage = "1v8";
+        };
+        /* configuration for UART0 default state */
+        uart1_rx_default: uart1_rx_default {
+                pinmuxs = <&pinctrlb 0 IT8XXX2_ALT_FUNC_3>;
+                bias-pull-up;
+        };
+        uart1_tx_default: uart1_tx_default {
+                pinmuxs = <&pinctrlb 1 IT8XXX2_ALT_FUNC_3>;
+        };
+      };
+
+    The 'uart1_rx_default' child node encodes the pin configurations
+    for a particular state of a device; in this case, the default
+    (that is, active) sate.
+
+    To link pin configurations with a device, use a pinctrl-N property for some
+    number N, like this example you could place in your board's DTS file:
+
+      #include "board-pinctrl.dtsi"
+
+      &uart0 {
+        pinctrl-0 = <&uart1_rx_default &uart1_tx_default>;
+        pinctrl-1 = <&uart1_rx_sleep &uart1_tx_sleep>;
+        pinctrl-2 = <&uart1_rx_priv &uart1_tx_priv>;
+        pinctrl-names = "default", "sleep", "priv_start";
+      };
+
+compatible: "ite,it8xxx2-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node.yaml
+      child-binding:
+        property-allowlist:
+          - bias-high-impedance
+          - bias-pull-pin-default
+          - bias-pull-up
+          - bias-pull-down
+          - input-enable
+
+child-binding:
+    description: |
+        This binding gives a base representation of the ITE IT8XXX2 pins configration.
+    properties:
+        pinmuxs:
+          required: true
+          type: phandle-array
+          description: |
+              ITE IT8XXX2 pin's configuration (pinctrl node, pin and function).
+
+        gpio-voltage:
+          required: false
+          type: string
+          description: |
+              Input voltage selection 3.3V or 1.8V
+          default: "3v3"
+          enum:
+          - "3v3"       # Default voltage.
+          - "1v8"
```

Comment:
defaults need to be justified, per guidelines. e.g. is this register default? Documentation needs to go into `description` field, not as a side comment.

Reply:
Done.